lerobot/common/policies/pretrained.py

```diff
@@ -42,6 +44,92 @@
 - Docs: {{ docs_url | default("[More Information Needed]", true) }}
 """
 
+# Matches ".soNNN", optionally followed by "-something", up to the "_buffer_" marker
+_VARIANT_RE = re.compile(r"\.so\d+(?:-[\w]+)?_buffer_")
+
+
+def canonicalise(k: str) -> str:
+    """
+    Remove dataset-variant markers like '.so100-blue_' or '.so100_' from a
+    normalisation-buffer key.
+    """
+    return _VARIANT_RE.sub(".buffer_", k)
+
+
+def standardise_state_dict(
+    ckpt: Dict[str, torch.Tensor], ref_keys: set[str], *, verbose: bool = True
```

Comment:
I prefer explicit variables
```suggestion
    checkpoint: Dict[str, torch.Tensor], ref_keys: set[str], *, verbose: bool = True
```